app/Http/Requests/Projects/CreateRequest.php

```diff
@@ -24,15 +24,25 @@ public function authorize()
      */
     public function rules()
     {
-        return [
+        $rules = [
             'name'           => 'required|max:50',
             'proposal_date'  => 'nullable|date|date_format:Y-m-d',
             'proposal_value' => 'nullable|numeric',
             'customer_id'    => 'nullable|numeric',
             'customer_name'  => 'nullable|required_without:customer_id|max:60',
             'customer_email' => 'nullable|required_without:customer_id|email|unique:users,email',
-            'description'    => 'nullable|max:255',
         ];
+
+        //Allow for flexibility instead of optionless hard-coded value for "description". This is
+        //achieved using environmental variable.
+        //A value of zero (0) will mean "no limit"
+
+        $char_len_project_description = intval(env('CHAR_LEN_PROJECT_DESCRIPTION', 255));
```

Comment:
I believe we just need 1 new env variable here, change it to just: **DESCRIPTION_CHAR_LEN**. Add this to **.env.example** file. And please use **camelCase** variable names.